MAINTAINERS

```diff
@@ -0,0 +1,68 @@
+[Org]
+
+	[Org."Core maintainers"]
+
+	# Current active contributors to the project.
+
+		people = [
+			"aaronprindle",
+			"danlorenc",
+			"mattrickard",
+			"nkubala"
+		]
+
+	[Org."Alumni"]
+
+	# People who have contributed to the project in the past.
+
+		people = [
+			"abbytisdale",
```

Comment:
I think this should be abbytiz if it's supposed to by my github username :)

Reply:
It's actually just the key into the `people` references section below, which has your name, email, and username. Check those and make sure they're right though :+1:

Reply:
Oh, LGTM